apps/api/Dockerfile

```diff
@@ -66,7 +64,5 @@ RUN --mount=type=cache,id=pnpm-store-api,target=/root/.pnpm-store\
  --frozen-lockfile \
  --unsafe-perm
 
-ENV NEW_RELIC_NO_CONFIG_FILE=true
-
 WORKDIR /usr/src/app/apps/api
-CMD [ "pm2-runtime","start", "dist/main.js" ]
+CMD [ "dump-init", "node", "dist/main.js" ]
```

Comment:
Should this one be `dumb-init`?

Reply:
I followed some best practises and found out about https://github.com/Yelp/dumb-init. Maybe we need it, maybe not. We will review it as soon as this is more testable.